Automattic/jetpack
# AI Assistant: Add usage panel scaffolding

Part of #33359.

## Proposed changes:

* Add a placeholder panel to hold usage information in next iterations
* Divide the AI Assistant sidebar panel using sections, put the content feedback tool under "AI feedback on post" and the usage panel under "Usage" (accoding to Figma styles from p1HpG7-oMv-p2)

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* With beta extensions enabled, go to the block editor, open the Jetpack sidebar and look for the AI Assistant section; it should contain the new sections division and a placeholder text for the Usage Panel:

<img width="351" alt="Screenshot 2023-10-18 at 18 33 50" src="https://github.com/Automattic/jetpack/assets/6760046/c7d9f06c-8fe5-47e1-9a76-52f9760480e7">

* With the beta extensions disabled, check the sidebar again and confirm the "Usage" section is not present, but the "AI feedback on post" is still there and working as expected:

<img width="342" alt="Screenshot 2023-10-18 at 18 35 03" src="https://github.com/Automattic/jetpack/assets/6760046/343212a9-11d7-4a59-a582-cd6d2b4f7771">
